_maps/cyberiad.dm

```diff
@@ -3,20 +3,19 @@ All z-levels should be identical in size. Their numbers should not matter.
 The order of z-levels should not matter as long as their attributes are properly defined at MAP_TRANSITION_CONFIG.
 Old code checked for the number of the z-level (for example whether there are any revheads on Z1),
```

Comment:
Can most likely be removed since I can't find an instance where a numeric or define value is checked

Reply:
Why remove it? It says what the code used to do, and why you should use the new system.

Reply:
It would suggest that there is still old code which checks it

Reply:
`checked` is past tense, not `checks`.